### DISCUSSION TITLE
Installing Homebrew on a Windows OS

### DISCUSSION THREAD

**Discussion by uoooosuf** [Homebrew/discussions#72] [Everyday usage]
I have being trying to install Homebrew on Windows OS, but anytime i tried to run the installation command I get an Error Telling me that: "The system cannot find the path specified".
Below is the code and the error message: 
C:\WINDOWS\system32>/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
The system cannot find the path specified.

**Comment by jonchang** [comment]
Homebrew doesn't work on Windows natively; you'll need to [set up Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/), then in a WSL shell [install Homebrew on Linux the usual way](https://docs.brew.sh/Homebrew-on-Linux).

**Reply by Akylzhan** [reply]
But out of curiosity, is it possible for Homebrew to work on Windows natively?

**Reply by SMillerDev** [reply]
Theoretically? Yes, you'll just need to rewrite any command invocations with a windows compatible version in the source. But then you basically invented https://chocolatey.org/

**Reply by chobotnice007** [reply]
> Homebrew doesn't work on Windows natively; you'll need to [set up Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/), then in a WSL shell [install Homebrew on Linux the usual way](https://docs.brew.sh/Homebrew-on-Linux).

Windows Subsystem for Linux is Linux in Windows. Windows isn't supported.
